docs/en/trust.rst

```diff
@@ -2,56 +2,298 @@
+
+The infrastructure of Trust enables the trust assessment mechanism to be applied between the parties defined in the `EIDAS-ARF`_.
+
+..  figure:: ../../images/trust-roles.svg
+    :alt: federation portrain
+    :width: 100%
     
+    The roles of the Federation infrastructure, where a Trust Anchor has one or more Intermediates and Leafs.
+
 General Properties
 ------------------
-    
-[TODO]
 
-    
-Requirements
-------------
+OpenID Federation facilitates the building of an infrastructure that is:
 
- - req 1
- - req 2
+- **Secure and Tamper-proof**, ensuring that entities' attestations of metadata and keys are cryptographically signed in the chain of trust, comprised of attestations issued by multiple parties that cannot be forged or tampered with by an adversary;
+- **Privacy-preserving**, as the infrastructure is public and only exposes public data such as public keys and metadata of the participants for interoperability needs. It does not require authentication of the requesters and therefore does not track who is assessing trust against whom;
+- **Guarantor of the non-repudiation of long-lived attestations**, historical keys endpoints and historical Trust Chains are saved for years according to data retention policies. This enables the certification of the validity of historical compliance, even in cases of revocation, expiration, or rotation of the keys used for signature verification;
+- **Dynamic and flexible**, allowing any participant to modify parts of their metadata autonomously, as these are published within their domains and verified through the Trust Chain. Simultaneously, the Trust Anchor or its Intermediate may publish a metadata policy to dynamically modify the metadata of all participants—such as disabling a vulnerable signature algorithm—and obtain certainty of propagation within a configured period of time to all participants;
+- **Efficient**, as JWT and JSON formats have been adopted on the web for years. They are cost-effective in terms of storage and processing and have a wide range of solutions available, such as libraries and software development kits, which enable rapid implementation of the solution;
+- **Scalable**, the Trust Model can accommodate more than a single organization by using Intermediates;
+- **Simple**, as it is based on widely used REST technology and formats that have become popular over the years.
 
+This implementation profile
+---------------------------
 
-Attributes
-----------
+This document applies the OpenID Connect Federation in its original state, without any substantive changes.
 
-[Table with parameters/attributes]
-    
-.. list-table:: 
-   :widths: 20 60
-   :header-rows: 1
+This document distinguishes between Federation Entities, which include all participants except for Wallet Instances.
```

Comment:
we should map the entities defined in the OIDC-FED with the entities in the EUDIW ecosystem. For RP (I assume here that Relying Parties are the same as Verifier, isn't it?) is easy, we already have this map. For Issuers, for example?
Moreover, while in CIE SPID OIDC an intermediate operates only for RP, here we could have different kind of intermediates (for issuer and for verifier). Distinguishing them could make sense. Last point, I know that the wallet instance is not considered as a participant, this implies that the wallet provider is a leaf. How can be defined within the OIDC-FED? Final remark: we should also handle a Verifier Instance in a similar way we manage a Wallet Instance. We should think about this.

Reply:
for RP we have 

`openid_relying_party` and `client` according to OpenID4VP that has defined client_metadata (too much wide ...)

for issuance we have `openid-credential-issuer`

for wallet provider we have proposed with @balanza and @grausof `eidas_wallet_provider` but we may go just with `wallet_provider`

Reply:
I'll add this table with mapping of eudi wallet ecosystem in federation configurations

Reply:
When you say we have `openid-credential-issuer`, exactly where is defined? The name implies that only openID Credential Issuers could exist. For credential Issuers not following the OpenID specs?

Reply:
It Is defined in openid4vci

Reply:
The only reference I see is in the URI suffix of the well-known  definition. Are we implicitly saying that the suffix of the well-known uri is also used as an entity role definition as metadata type identifier in the context of OIDC-FED? They are semantically different I think.

Reply:
I didn't understand, what do you mean?

Reply:
You are saying to use `openid-credential-issuer` as an entity type identifier for the Issuers, right? 
Basing on https://openid.net/specs/openid-connect-federation-1_0.html#name-metadata-type-identifiers we don't have this identifier and neither in the OIDC4VCI (we have only as an URI suffix of the well-known which has a different semantic). So my question is: What you are proposing is something new (and then we have to define somewhere here) or is taken from some other spec?

Reply:
OIDC Federation allow the definition of any new kind of entity

![immagine](https://github.com/italia/eidas-it-wallet-docs/assets/1297620/4f223454-8599-46b7-a968-1c0f6739a73b)